.gitignore

```diff
@@ -6,3 +6,4 @@ result
 /include
 /lib
 pyvenv.cfg
+/.direnv
```

Comment:
Somewhat unrelated change, but I was getting a dirty worktree without this while iterating.